src/configure.js

```diff
@@ -1,6 +1,6 @@
 import path from 'path'
 
-export function configure() {
+export default function configure() {
```

Comment:
No need for naming the function since it's a default one

```suggestion
export default function () {
```

Reply:
Named functions make it easy to see the stacktrace